stable/awsebscsiprovisioner/values.yaml

```diff
@@ -66,3 +66,10 @@ storageclass:
   reclaimPolicy: Delete
   volumeBindingMode: WaitForFirstConsumer
```

Comment:
I'm no expert in topologies, but according to https://kubernetes.io/docs/concepts/storage/storage-classes/#allowed-topologies it isn't necessary to set allowed topologies in the `WaitForFirstConsumer` case. Is this added to support the `Immediate` case as well?

Reply:
There is some confusion because one is not defining any restrictions and assumes the volume can be mounted whenever a pod asks for it, and or relying on whether the pod already defined restrictions.
 
Meanwhile when you use allowedTopologies, it is you who set these volume affinity policies to any pod that uses this storage class.  One doesn't exclude the other.

Reply:
As an example, WaitForFirstConsumer without topologies will set the AZ to the AZ of the instance that asked for the pod. However I might not want to have that behavior and restrict that scenario to a specific or multiple AZ different than the one defined by the instance that claimed that vol. Therefore you can add allowedTopologies and awsebssi will pick (that is very unfortunate) the [0] AZ from that list.

Reply:
So the pod will be forced to be scheduled in an AZ where it can use the volumes by specified in the allowedTopologies